html/fonts/icomoon.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Can you leave a small README in the fonts folder explaining what icomoon is, and how to modify it? Just 2-3 lines should be fine.